src/features/search/pages/Search/Search.native.test.tsx

```diff
@@ -201,6 +202,8 @@ jest.mock('libs/location/LocationWrapper', () => ({
   }),
 }))
 
+jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(true).mockReturnValue(true)
```

Comment:
```suggestion
jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(true)
```
Je crois qu'il y a un doublon